views/homepage.handlebars

```diff
@@ -1,9 +1,19 @@
-<section class='container text-center'>
-  <h2>Welcome{{#if loggedIn}} {{userData.username}}{{/if}}!</h2>
-  <p>
-    Welcome to a growing community of coders working hand-in-hand to make sure
-    no buddy gets left behind
-  </p>
+<div class='cover text-center'>
+  <img id='studying-img' src='personstudying.png' />
+  <img id='studying-logo' src='studdybuddytitle.png' /></div>
```

Comment:
this src wont work. has to be relative and pointing at image folder so something like ./images/file